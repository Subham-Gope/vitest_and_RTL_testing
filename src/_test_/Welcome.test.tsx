// import { describe, test, expect } from "vitest";
// import { render, screen } from "@testing-library/react";
// import Welcome from "../components/Welcome";

// describe("testing the Welcome.tsx", () => {
//   test("renders the welcome message", () => {
//     const { container } = render(<Welcome />);

//     expect(container).toMatchSnapshot();

//     const message = screen.getByText("Welcome to React Testing");
//     expect(message).toBeInTheDocument();
//   });
// });
