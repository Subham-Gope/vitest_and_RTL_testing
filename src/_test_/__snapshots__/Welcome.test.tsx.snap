// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`testing the Welcome.tsx > renders the welcome message 1`] = `
<div>
  <h1>
    Welcome to React Testing
  </h1>
</div>
`;
